diff --git a/win64.mak b/win64.mak
index 9746daf..a35184b 100644
--- a/win64.mak
+++ b/win64.mak
@@ -26,10 +26,6 @@ CP=cp
 
 DIR=\dmd2
 
-## Visual C directories
-VCDIR=\Program Files (x86)\Microsoft Visual Studio 10.0\VC
-SDKDIR=\Program Files (x86)\Microsoft SDKs\Windows\v7.0A
-
 ## Flags for VC compiler
 
 #CFLAGS=/Zi /nologo /I"$(VCDIR)\INCLUDE" /I"$(SDKDIR)\Include"
@@ -47,9 +43,9 @@ UDFLAGS=-g -m$(MODEL) -O -w -d
 
 ## C compiler, linker, librarian
 
-CC="$(VCDIR)\bin\amd64\cl"
-LD="$(VCDIR)\bin\amd64\link"
-AR="$(VCDIR)\bin\amd64\lib"
+CC="$(VCBIN_DIR)\cl"
+LD="$(VCBIN_DIR)\link"
+AR="$(VCBIN_DIR)\lib"
 MAKE=make
 
 ## D compiler
