diff --git a/win64.mak b/win64.mak
index daee7f6..b381a34 100644
--- a/win64.mak
+++ b/win64.mak
@@ -2,14 +2,11 @@

 MODEL=64

-VCDIR=\Program Files (x86)\Microsoft Visual Studio 10.0\VC
-SDKDIR=\Program Files (x86)\Microsoft SDKs\Windows\v7.0A
-
 DMD=dmd

-CC="$(VCDIR)\bin\amd64\cl"
-LD="$(VCDIR)\bin\amd64\link"
-AR="$(VCDIR)\bin\amd64\lib"
+CC="$(VCDIR)\bin\x86_amd64\cl"
+LD="$(VCDIR)\bin\x86_amd64\link"
+AR="$(VCDIR)\bin\x86_amd64\lib"
 CP=cp

 DOCDIR=doc

