diff --git a/etc/c/zlib/win64.mak b/etc/c/zlib/win64.mak
index 1edfd0e..7e6d8ee 100644
--- a/etc/c/zlib/win64.mak
+++ b/etc/c/zlib/win64.mak
@@ -1,10 +1,8 @@
 # Makefile for zlib64

-VCDIR=\Program Files (x86)\Microsoft Visual Studio 10.0\VC
-
-CC="$(VCDIR)\bin\amd64\cl"
-LD="$(VCDIR)\bin\amd64\link"
-LIB="$(VCDIR)\bin\amd64\lib"
+CC="$(VCBIN_DIR)\cl"
+LD="$(VCBIN_DIR)\link"
+LIB="$(VCBIN_DIR)\lib"

 CFLAGS=/O2 /nologo /I"$(VCDIR)\INCLUDE"
 LIBFLAGS=/nologo
diff --git a/win64.mak b/win64.mak
index 9746daf..a35184b 100644
--- a/win64.mak
+++ b/win64.mak
@@ -26,10 +26,6 @@ CP=cp
 
 DIR=\dmd2
 
-## Visual C directories
-VCDIR=\Program Files (x86)\Microsoft Visual Studio 10.0\VC
-SDKDIR=\Program Files (x86)\Microsoft SDKs\Windows\v7.0A
-
 ## Flags for VC compiler
 
 #CFLAGS=/Zi /nologo /I"$(VCDIR)\INCLUDE" /I"$(SDKDIR)\Include"
@@ -47,9 +43,9 @@ UDFLAGS=-g -m$(MODEL) -O -w -d
 
 ## C compiler, linker, librarian
 
-CC="$(VCDIR)\bin\amd64\cl"
-LD="$(VCDIR)\bin\amd64\link"
-AR="$(VCDIR)\bin\amd64\lib"
+CC="$(VCBIN_DIR)\cl"
+LD="$(VCBIN_DIR)\link"
+AR="$(VCBIN_DIR)\lib"
 MAKE=make
 
 ## D compiler
@@ -447,7 +443,7 @@ html : $(DOCS)
 
 $(ZLIB): $(SRC_ZLIB)
 	cd etc\c\zlib
-	$(MAKE) -f win$(MODEL).mak zlib$(MODEL).lib "CC=\$(CC)"\"" "LIB=\$(AR)"\"" "VCDIR=$(VCDIR)"
+	$(MAKE) -f win$(MODEL).mak zlib$(MODEL).lib "CC=\$(CC)"\"" "LIB=\$(AR)"\""
 	cd ..\..\..
 
 ################## DOCS ####################################
